openff/qcsubmit/results/results.py

```diff
@@ -218,6 +219,39 @@ def filter(self, *filters: "ResultFilter"):
 
         return filtered_collection
 
+    def visualize(
+        self,
+        file_name: str,
+        columns: int = 4,
+        toolkit: Optional[Literal["openeye", "rdkit"]] = None,
+    ) -> None:
+        """
+        Create a pdf file of the molecules referenced by this collection using either
+        OpenEye or RDKit.
+
+        Parameters:
+            file_name: The name of the pdf file which will be produced.
+            columns: The number of molecules per row.
+            toolkit: The backend toolkit to use when producing the pdf file.
+        """
+
+        # We filter by inchi key to make sure that we don't double count molecules
+        # with different orderings.
+        unique_smiles = set(
```

Comment:
would it be useful to add an API point for the list of unique smiles, I think I have seen this on a few PRs.

Reply:
Good call! I'll handle this in a separate PR after this is merged.